### PR TITLE
bambenek mapping

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
added mapping description to malwarefamily

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: Deleted all outputs of the get-indicators command.
   - [ ] No

### CODE REVIEW

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ configuration:
   required: false
```
Comment (avidan-H) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml]:
```suggestion
  required: false
  defaultvalue: true
```

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ configuration:
 - additionalinfo: Indicators from this integration instance will be marked with this
     reputation.
```
Comment (avidan-H) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml]:
```suggestion
    reputation
```

File: Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml [hunk 3]
```diff
@@ -17,12 +17,12 @@ configuration:
   required: false
   type: 8
 - defaultvalue: '360'
-  display: Fetch Interval
+  display: Feed Fetch Interval
   name: feedFetchInterval
   required: false
   type: 19
 - defaultvalue: B - Usually reliable
```
Comment (avidan-H) [inline Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml]:
```suggestion
- defaultvalue: B - Usually reliable
  additionalinfo: Reliability of the source providing the intelligence data
```

**Comment by Itay4** [comment]
force merging as the build fails on BC checks on context outputs removed according to the new feed design
the non-bc build passes

**Pull request merged**